lxml build fails against Python 3.13

## Expected

`brew install saulpw/vd/visidata` succeeds

## Actual

```
brew install saulpw/vd/visidata
...
==> Installing visidata from saulpw/vd
==> python3 -m venv --system-site-packages --without-pip /opt/homebrew/Cellar/visidata/3.0.2_1/libexec
==> python3 -m pip --python=/opt/homebrew/Cellar/visidata/3.0.2_1/libexec/bin/python install /private/tmp/visidata--six-20241014-13614-11396b/six-1.16.0
==> python3 -m pip --python=/opt/homebrew/Cellar/visidata/3.0.2_1/libexec/bin/python install /private/tmp/visidata--python-dateutil-20241014-13614-3jzrmv/python-dateutil-2.8.2
==> python3 -m pip --python=/opt/homebrew/Cellar/visidata/3.0.2_1/libexec/bin/python install /private/tmp/visidata--et-xmlfile-20241014-13614-1a65if/et_xmlfile-1.1.0
==> python3 -m pip --python=/opt/homebrew/Cellar/visidata/3.0.2_1/libexec/bin/python install /private/tmp/visidata--jdcal-20241014-13614-im56p1/jdcal-1.4.1
==> python3 -m pip --python=/opt/homebrew/Cellar/visidata/3.0.2_1/libexec/bin/python install /private/tmp/visidata--openpyxl-20241014-13614-f7x648/openpyxl-3.0.10
==> python3 -m pip --python=/opt/homebrew/Cellar/visidata/3.0.2_1/libexec/bin/python install /private/tmp/visidata--xlrd-20241014-13614-uyvw0k/xlrd-2.0.1
==> python3 -m pip --python=/opt/homebrew/Cellar/visidata/3.0.2_1/libexec/bin/python install /private/tmp/visidata--chardet-20241014-13614-p2qhg1/chardet-5.1.0
==> python3 -m pip --python=/opt/homebrew/Cellar/visidata/3.0.2_1/libexec/bin/python install /private/tmp/visidata--lxml-20241014-13614-9wwk2q/lxml-4.9.4
Last 15 lines from /Users/swilson/Library/Logs/Homebrew/visidata/09.python3:
  cc private/tmp/xmlXPathInith6mnhdea.o -L/Library/Developer/CommandLineTools/SDKs/MacOSX14.sdk/usr/lib -lxml2 -o a.out
  error: command '/opt/homebrew/Library/Homebrew/shims/mac/super/clang' failed with exit code 1
  error: subprocess-exited-with-error

  × Building wheel for lxml (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.

  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: /opt/homebrew/Cellar/visidata/3.0.2_1/libexec/bin/python /opt/homebrew/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py build_wheel /private/tmp/tmpi76xcd6w
  cwd: /private/tmp/visidata--lxml-20241014-13614-9wwk2q/lxml-4.9.4
  Building wheel for lxml (pyproject.toml): finished with status 'error'
  ERROR: Failed building wheel for lxml
Failed to build lxml
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (lxml)
```

## More info

The error output from clang:

```
2024-10-14 22:15:50 +0000

python3
-m
pip
--python=/opt/homebrew/Cellar/visidata/3.0.2_1/libexec/bin/python
install
--verbose
--no-deps
--no-binary=:all:
--ignore-installed
--no-compile
/private/tmp/visidata--lxml-20241014-13614-9wwk2q/lxml-4.9.4

...
  running build_ext
  building 'lxml.etree' extension
  creating build/temp.macosx-14.0-arm64-cpython-313/src/lxml
  clang -fno-strict-overflow -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -O3 -Wall -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.sdk -DCYTHON_CLINE_IN_TRACEBACK=0 -I/Library/Developer/CommandLineTools/SDKs/MacOSX14.sdk/usr/include -Isrc -Isrc/lxml/includes -I/opt/homebrew/Cellar/visidata/3.0.2_1/libexec/include -I/opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13 -c src/lxml/etree.c -o build/temp.macosx-14.0-arm64-cpython-313/src/lxml/etree.o -w -flat_namespace
  src/lxml/etree.c:5988:39: error: call to undeclared function '_PyInterpreterState_GetConfig'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
   5988 |     __pyx_assertions_enabled_flag = ! _PyInterpreterState_GetConfig(__Pyx_PyThreadState_Current->interp)->optimization_level;
        |                                       ^
  src/lxml/etree.c:5988:107: error: member reference type 'int' is not a pointer
   5988 |     __pyx_assertions_enabled_flag = ! _PyInterpreterState_GetConfig(__Pyx_PyThreadState_Current->interp)->optimization_level;
        |                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  ^
  src/lxml/etree.c:255376:7: error: call to undeclared function '_PyDict_SetItem_KnownHash'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
   255376 |   if (__Pyx_SetNameInClass(__pyx_t_2, __pyx_n_s_getitem, __pyx_t_9) < 0) __PYX_ERR(0, 97, __pyx_L1_error)
          |       ^
  src/lxml/etree.c:6652:38: note: expanded from macro '__Pyx_SetNameInClass'
   6652 |     (likely(PyDict_CheckExact(ns)) ? _PyDict_SetItem_KnownHash(ns, name, value, ((PyASCIIObject *) name)->hash) : PyObject_SetItem(ns, name, value))
        |                                      ^
  src/lxml/etree.c:255376:7: note: did you mean '_PyDict_GetItem_KnownHash'?
  src/lxml/etree.c:6652:38: note: expanded from macro '__Pyx_SetNameInClass'
   6652 |     (likely(PyDict_CheckExact(ns)) ? _PyDict_SetItem_KnownHash(ns, name, value, ((PyASCIIObject *) name)->hash) : PyObject_SetItem(ns, name, value))
        |                                      ^
  /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13/cpython/dictobject.h:38:24: note: '_PyDict_GetItem_KnownHash' declared here
     38 | PyAPI_FUNC(PyObject *) _PyDict_GetItem_KnownHash(PyObject *mp, PyObject *key,
        |                        ^
  src/lxml/etree.c:264446:13: error: call to undeclared function '_PyUnicode_FastCopyCharacters'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
   264446 |             _PyUnicode_FastCopyCharacters(result_uval, char_pos, uval, 0, ulength);
          |             ^
  src/lxml/etree.c:264446:13: note: did you mean 'PyUnicode_CopyCharacters'?
  /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13/cpython/unicodeobject.h:416:24: note: 'PyUnicode_CopyCharacters' declared here
    416 | PyAPI_FUNC(Py_ssize_t) PyUnicode_CopyCharacters(
        |                        ^
  src/lxml/etree.c:265182:35: error: use of undeclared identifier '_PyObject_NextNotImplemented'; did you mean 'PyObject_HashNotImplemented'?
   265182 |         if (unlikely(iternext == &_PyObject_NextNotImplemented))
          |                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
          |                                   PyObject_HashNotImplemented
  src/lxml/etree.c:1096:43: note: expanded from macro 'unlikely'
   1096 |   #define unlikely(x) __builtin_expect(!!(x), 0)
        |                                           ^
  /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13/object.h:591:23: note: 'PyObject_HashNotImplemented' declared here
    591 | PyAPI_FUNC(Py_hash_t) PyObject_HashNotImplemented(PyObject *);
        |                       ^
  src/lxml/etree.c:265973:13: error: call to undeclared function '_PyGen_SetStopIterationValue'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
   265973 |             _PyGen_SetStopIterationValue(result);
          |             ^
  src/lxml/etree.c:265973:13: note: did you mean '__Pyx_PyGen__FetchStopIterationValue'?
  src/lxml/etree.c:265728:12: note: '__Pyx_PyGen__FetchStopIterationValue' declared here
   265728 | static int __Pyx_PyGen__FetchStopIterationValue(CYTHON_UNUSED PyThreadState *__pyx_tstate, PyObject **pvalue) {
          |            ^
  src/lxml/etree.c:267806:5: error: call to undeclared function '_PyErr_FormatFromCause'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
   267806 |     _PyErr_FormatFromCause(
          |     ^
  src/lxml/etree.c:268071:19: error: call to undeclared function '_PySet_NextEntry'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
   268071 |         int ret = _PySet_NextEntry(iter_obj, ppos, value, &hash);
          |                   ^
  src/lxml/etree.c:269068:70: error: too few arguments to function call, expected 6, have 5
   269066 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
          |                           ~~~~~~~~~~~~~~~~~~~
   269067 |                                               bytes, sizeof(val),
   269068 |                                               is_little, !is_unsigned);
          |                                                                      ^
  /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13/cpython/longobject.h:111:17: note: '_PyLong_AsByteArray' declared here
    111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
        |                 ^                   ~~~~~~~~~~~~~~~~
    112 |     unsigned char* bytes, size_t n,
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    113 |     int little_endian, int is_signed, int with_exceptions);
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  src/lxml/etree.c:269377:70: error: too few arguments to function call, expected 6, have 5
   269375 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
          |                           ~~~~~~~~~~~~~~~~~~~
   269376 |                                               bytes, sizeof(val),
   269377 |                                               is_little, !is_unsigned);
          |                                                                      ^
  /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13/cpython/longobject.h:111:17: note: '_PyLong_AsByteArray' declared here
    111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
        |                 ^                   ~~~~~~~~~~~~~~~~
    112 |     unsigned char* bytes, size_t n,
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    113 |     int little_endian, int is_signed, int with_exceptions);
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  src/lxml/etree.c:269573:70: error: too few arguments to function call, expected 6, have 5
   269571 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
          |                           ~~~~~~~~~~~~~~~~~~~
   269572 |                                               bytes, sizeof(val),
   269573 |                                               is_little, !is_unsigned);
          |                                                                      ^
  /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13/cpython/longobject.h:111:17: note: '_PyLong_AsByteArray' declared here
    111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
        |                 ^                   ~~~~~~~~~~~~~~~~
    112 |     unsigned char* bytes, size_t n,
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    113 |     int little_endian, int is_signed, int with_exceptions);
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  src/lxml/etree.c:269769:70: error: too few arguments to function call, expected 6, have 5
   269767 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
          |                           ~~~~~~~~~~~~~~~~~~~
   269768 |                                               bytes, sizeof(val),
   269769 |                                               is_little, !is_unsigned);
          |                                                                      ^
  /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13/cpython/longobject.h:111:17: note: '_PyLong_AsByteArray' declared here
    111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
        |                 ^                   ~~~~~~~~~~~~~~~~
    112 |     unsigned char* bytes, size_t n,
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    113 |     int little_endian, int is_signed, int with_exceptions);
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  src/lxml/etree.c:270003:70: error: too few arguments to function call, expected 6, have 5
   270001 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
          |                           ~~~~~~~~~~~~~~~~~~~
   270002 |                                               bytes, sizeof(val),
   270003 |                                               is_little, !is_unsigned);
          |                                                                      ^
  /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13/cpython/longobject.h:111:17: note: '_PyLong_AsByteArray' declared here
    111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
        |                 ^                   ~~~~~~~~~~~~~~~~
    112 |     unsigned char* bytes, size_t n,
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    113 |     int little_endian, int is_signed, int with_exceptions);
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  src/lxml/etree.c:270237:70: error: too few arguments to function call, expected 6, have 5
   270235 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
          |                           ~~~~~~~~~~~~~~~~~~~
   270236 |                                               bytes, sizeof(val),
   270237 |                                               is_little, !is_unsigned);
          |                                                                      ^
  /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13/cpython/longobject.h:111:17: note: '_PyLong_AsByteArray' declared here
    111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
        |                 ^                   ~~~~~~~~~~~~~~~~
    112 |     unsigned char* bytes, size_t n,
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    113 |     int little_endian, int is_signed, int with_exceptions);
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  src/lxml/etree.c:270440:70: error: too few arguments to function call, expected 6, have 5
   270438 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
          |                           ~~~~~~~~~~~~~~~~~~~
   270439 |                                               bytes, sizeof(val),
   270440 |                                               is_little, !is_unsigned);
          |                                                                      ^
  /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13/cpython/longobject.h:111:17: note: '_PyLong_AsByteArray' declared here
    111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
        |                 ^                   ~~~~~~~~~~~~~~~~
    112 |     unsigned char* bytes, size_t n,
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    113 |     int little_endian, int is_signed, int with_exceptions);
        |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  15 errors generated.
  Compile failed: command '/opt/homebrew/Library/Homebrew/shims/mac/super/clang' failed with exit code 1
  creating private/tmp
  cc -I/Library/Developer/CommandLineTools/SDKs/MacOSX14.sdk/usr/include -I/usr/include/libxml2 -c /private/tmp/xmlXPathInith6mnhdea.c -o private/tmp/xmlXPathInith6mnhdea.o
  cc private/tmp/xmlXPathInith6mnhdea.o -L/Library/Developer/CommandLineTools/SDKs/MacOSX14.sdk/usr/lib -lxml2 -o a.out
  error: command '/opt/homebrew/Library/Homebrew/shims/mac/super/clang' failed with exit code 1
  error: subprocess-exited-with-error
  
  × Building wheel for lxml (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
```

## Workaround

Changing the formula to specifically ask for 3.12 worked:

```diff
--- /opt/homebrew/Library/Taps/saulpw/homebrew-vd/Formula/visidata.rb	2024-10-14 15:33:57
+++ /opt/homebrew/Library/Taps/saulpw/homebrew-vd/Formula/visidata.rb	2024-10-14 15:23:42
@@ -6,7 +6,7 @@
   sha256 "38c38df08c1b48dde5f9ae1bdb02cb19f70acf0461d0e85cb14f9c002a77c05a"
   revision 1
 
-  depends_on "python"
+  depends_on "python@3.12"
 
   on_linux do
     depends_on "libxml2"

```

but perhaps upgrading lxml would be preferred, as was done in https://github.com/saulpw/visidata/issues/2329#issuecomment-1965912070

--- a/Formula/visidata.rb
+++ b/Formula/visidata.rb
@@ -2,8 +2,8 @@ class Visidata < Formula
   include Language::Python::Virtualenv
   desc "Terminal utility for exploring and arranging tabular data"
   homepage "https://visidata.org/"
-  url "https://files.pythonhosted.org/packages/8c/75/3e2f5c7c8b129d34a28d02741e13ae6e1bc41c3fcdb0a3e45b54eb2defe0/visidata-3.0.2.tar.gz"
-  sha256 "38c38df08c1b48dde5f9ae1bdb02cb19f70acf0461d0e85cb14f9c002a77c05a"
+  url "https://files.pythonhosted.org/packages/6c/41/b026336b7075a2c8461c1ba0dfffd456902f736722f4a016f21fee1e6d7c/visidata-3.1.1.tar.gz"
+  sha256 "8c4484158f0851e4887e595542bd65bdd991de9c20652b442b3fd9742ce8b031"
   revision 1
 
   depends_on "python3"
@@ -19,13 +19,13 @@ class Visidata < Formula
   end
 
   resource "python-dateutil" do
-    url "https://files.pythonhosted.org/packages/4c/c4/13b4776ea2d76c115c1d1b84579f3764ee6d57204f6be27119f13a61d0a9/python-dateutil-2.8.2.tar.gz"
-    sha256 "0123cacc1627ae19ddf3c27a5de5bd67ee4586fbdd6440d9748f8abb483d3e86"
+    url "https://files.pythonhosted.org/packages/d9/77/bd458a2e387e98f71de86dcc2ca2cab64489736004c80bc12b70da8b5488/python-dateutil-2.9.0.tar.gz"
+    sha256 "78e73e19c63f5b20ffa567001531680d939dc042bf7850431877645523c66709"
   end
 
   resource "et-xmlfile" do
-    url "https://files.pythonhosted.org/packages/3d/5d/0413a31d184a20c763ad741cc7852a659bf15094c24840c5bdd1754765cd/et_xmlfile-1.1.0.tar.gz"
-    sha256 "8eb9e2bc2f8c97e37a2dc85a09ecdcdec9d8a396530a6d5a33b30b9a92da0c5c"
+    url "https://files.pythonhosted.org/packages/d3/38/af70d7ab1ae9d4da450eeec1fa3918940a5fafb9055e934af8d6eb0c2313/et_xmlfile-2.0.0.tar.gz"
+    sha256 "dab3f4764309081ce75662649be815c4c9081e88f0837825f90fd28317d4da54"
   end
 
   resource "jdcal" do
@@ -34,8 +34,8 @@ class Visidata < Formula
   end
 
   resource "openpyxl" do
-    url "https://files.pythonhosted.org/packages/2c/b8/ff77a718173fd73e49f883b4fda88f11af1fc51edb9252af3785b0cad987/openpyxl-3.0.10.tar.gz"
-    sha256 "e47805627aebcf860edb4edf7987b1309c1b3632f3750538ed962bbcc3bd7449"
+    url "https://files.pythonhosted.org/packages/3d/f9/88d94a75de065ea32619465d2f77b29a0469500e99012523b91cc4141cd1/openpyxl-3.1.5.tar.gz"
+    sha256 "cf0e3cf56142039133628b5acffe8ef0c12bc902d2aadd3e0fe5878dc08d1050"
   end
 
   resource "xlrd" do
@@ -44,28 +44,28 @@ class Visidata < Formula
   end
 
   resource "chardet" do
-    url "https://files.pythonhosted.org/packages/41/32/cdc91dcf83849c7385bf8e2a5693d87376536ed000807fa07f5eab33430d/chardet-5.1.0.tar.gz"
-    sha256 "0d62712b956bc154f85fb0a266e2a3c5913c2967e00348701b32411d6def31e5"
+    url "https://files.pythonhosted.org/packages/f3/0d/f7b6ab21ec75897ed80c17d79b15951a719226b9fababf1e40ea74d69079/chardet-5.2.0.tar.gz"
+    sha256 "1b3b6ff479a8c414bc3fa2c0852995695c4a026dcd6d0633b2dd092ca39c1cf7"
   end
 
   resource "lxml" do
-    url "https://files.pythonhosted.org/packages/84/14/c2070b5e37c650198de8328467dd3d1681e80986f81ba0fea04fc4ec9883/lxml-4.9.4.tar.gz"
-    sha256 "b1541e50b78e15fa06a2670157a1962ef06591d4c998b998047fff5e3236880e"
+    url "https://files.pythonhosted.org/packages/e7/6b/20c3a4b24751377aaa6307eb230b66701024012c29dd374999cc92983269/lxml-5.3.0.tar.gz"
+    sha256 "4e109ca30d1edec1ac60cdbe341905dc3b8f55b16855e03a54aaf59e51ec8c6f"
   end
 
   resource "zipp" do
-    url "https://files.pythonhosted.org/packages/58/03/dd5ccf4e06dec9537ecba8fcc67bbd4ea48a2791773e469e73f94c3ba9a6/zipp-3.17.0.tar.gz"
-    sha256 "84e64a1c28cf7e91ed2078bb8cc8c259cb19b76942096c8d7b84947690cabaf0"
+    url "https://files.pythonhosted.org/packages/54/bf/5c0000c44ebc80123ecbdddba1f5dcd94a5ada602a9c225d84b5aaa55e86/zipp-3.20.2.tar.gz"
+    sha256 "bc9eb26f4506fda01b81bcde0ca78103b6e62f991b381fec825435c836edbc29"
   end
 
   resource "importlib-resources" do
-    url "https://files.pythonhosted.org/packages/4e/a2/3cab1de83f95dd15297c15bdc04d50902391d707247cada1f021bbfe2149/importlib_resources-5.12.0.tar.gz"
-    sha256 "4be82589bf5c1d7999aedf2a45159d10cb3ca4f19b2271f8792bc8e6da7b22f6"
+    url "https://files.pythonhosted.org/packages/98/be/f3e8c6081b684f176b761e6a2fef02a0be939740ed6f54109a2951d806f3/importlib_resources-6.4.5.tar.gz"
+    sha256 "980862a1d16c9e147a59603677fa2aa5fd82b87f223b6cb870695bcfce830065"
   end
 
   resource "importlib-metadata" do
-    url "https://files.pythonhosted.org/packages/90/b4/206081fca69171b4dc1939e77b378a7b87021b0f43ce07439d49d8ac5c84/importlib_metadata-7.0.1.tar.gz"
-    sha256 "f238736bb06590ae52ac1fab06a3a9ef1d8dce2b7a35b5ab329371d6c8f5d2cc"
+    url "https://files.pythonhosted.org/packages/cd/12/33e59336dca5be0c398a7482335911a33aa0e20776128f038019f1a95f1b/importlib_metadata-8.5.0.tar.gz"
+    sha256 "71522656f0abace1d072b9e5481a48f07c138e00f079c38c8f883823f9c26bd7"
   end
 
   def install

--- a/README.md
+++ b/README.md
@@ -4,7 +4,7 @@ homebrew-vd is a [Homebrew](http://brew.sh) tap for convenient installation and 
 
 If you are looking for installation instructions for VisData on OSes in the Linux family, they are available [here](https://github.com/saulpw/visidata#install).
 
-This tap contains VisiData v3.0.2.
+This tap contains VisiData v3.1.1.
 
 ## Install
 

